bin/dcps_tests.lst

```diff
@@ -46,7 +46,7 @@ tests/DCPS/LivelinessTest/run_test.pl udp: !DCPS_MIN !DDS_NO_OWNERSHIP_PROFILE !
 tests/DCPS/LivelinessTest/run_test.pl rtps_disc: !DCPS_MIN !DDS_NO_OWNERSHIP_PROFILE RTPS
 tests/DCPS/LivelinessTest/run_test.pl rtps_disc take: !DCPS_MIN !DDS_NO_OWNERSHIP_PROFILE RTPS
 
-tests/DCPS/Inconsistent_Qos/run_test.pl: RTPS DDS4CCM_OPENDDS
+tests/DCPS/Inconsistent_Qos/run_test.pl: RTPS DDS4CCM_OPENDDS !STATIC
```

Comment:
Does this test need to be only run with DDS4CCM_OPENDDS?  It used to be that the QOS_XML_Handler library was only available when dds4ccm was enabled, but this is no longer the case.